README.md

```diff
@@ -635,6 +635,9 @@ let g:ale_keep_list_window_open = 1
 You can also set `let g:ale_list_vertical = 1` to open the windows vertically
 instead of the default horizontally.
 
+To present `ALEFindReferences` results in the quickfix window, use
+`let g:ale_use_quickfix_findref = 1`.
```

Comment:
Instead of making this a global option, make this an option for the command. Have a look at the existing option below, and uses of the `ale#args#Parse` in the codebase for other commands in `symbol.vim` and `loclist_jumping.vim`. We could make the option in this case `'quickfix'`, so we can use it like `:ALEFindReferences -quickfix`.